docs/alert/index.html

```diff
@@ -142,7 +142,7 @@ <h3 class="title">Code Snippets</h3>
 
           <!-- Outlined Alerts -->
           <h2 class="title h2 m-v-2 colored-bg">Outlined alerts</h2>
-          <div class="container colored-container col-flex">
+          <div class="container colored-container m-v-4 col-flex">
```

Comment:
I think naming for margin and padding classes could be better, rest of everything is good.

Reply:
inspired from tailwind CSS 😅 but, sure will try to rename it.